src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Resources/Create/CreateResourceValidatorTests.cs

```diff
@@ -91,5 +94,64 @@ public void GivenConfigOrHeader_WhenValidatingCreate_ThenProfileValidationShould
                 profileValidator.DidNotReceive().TryValidate(Arg.Any<ITypedElement>(), Arg.Any<string>());
             }
         }
+
+        [Theory]
+        [InlineData("")]
+        [InlineData("1+1")]
+        [InlineData("11|")]
+        [InlineData("00000000000000000000000000000000000000000000000000000000000000065")]
+        public void GivenAResourceWithInvalidId_WhenValidatingUpsert_ThenInvalidShouldBeReturnedWithErrors(string id)
```

Comment:
>GivenAResourceWithInvalidId_WhenValidatingUpsert_ThenInvalidShouldBeReturnedWithErrors [](http://example.com/codeflow?start=20&length=86)

can `GivenAResourceWithInvalidId_WhenValidatingUpsert_ThenInvalidShouldBeReturned`

be extended instead of new test?

Reply:
yes we can